{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -67,7 +67,8 @@ def make_specs(template: str, location: str, output_dir: str, verbose: int, with
               default='all',
               show_default=True,
               type=click.Choice(project_globals.LOCATIONS + ['all']),
-              help='Location to make artifact for.')
+              help=('Location to make artifact for. Note: specifying "all" requires '
```

Comment:
If not running with all, you need to be on an archive node with j drive access.

Reply:
Reworded.